client/components/CandidateTestPlanRun/CandidateTestPlanRun.css

```diff
@@ -0,0 +1,154 @@
+.card-body {
+    padding: 0;
+}
+
+.card-body > div {
+    border: none;
+}
+
+.current-test-title {
+    border: none;
+}
+
+.current-test-options .options-wrapper {
+    text-align: center;
+}
+
+.current-test-options .options-wrapper li {
+    margin-bottom: 0.5em;
+}
+
+.current-test-options .options-wrapper li:last-child {
+    margin-bottom: 0;
+}
+
+.feedback-accordion {
+    width: 100%;
+}
+
+.feedback-accordion-header {
+    display: grid;
+    grid-template-columns: 50% 50%;
+}
+
+.feedback-accordion-header span {
+    text-align: right;
+}
+
+.feedback-from-text {
+    font-weight: 400;
+}
+
+.feedback-list-item {
+    padding: 1em 0 1em 0.5em;
+    position: relative;
+    margin: 1em 0 1em 0em;
+    list-style: none;
+    font-size: 14px;
+    border: solid 1px #e5e5e5;
+    width: 100%;
+}
+
+.feedback-indicator {
+    position: absolute;
+    left: -2em;
+    top: 1.3em;
+    width: 16px;
+    height: 16px;
+    border-radius: 50px;
+    border: 2px solid #9a9a9a;
+}
+
+.feedback-list-item:before {
+    content: '';
+    position: absolute;
+    height: 2.3em;
+    width: 2px;
+    top: -1.1em;
+    left: -1.5em;
+    background: #d2d5d9;
+}
+
+.feedback-list-item:after {
+    content: '';
+    position: absolute;
+    height: calc(100% - 1.3em);
+    width: 2px;
+    bottom: -1.2em;
+    left: -1.5em;
+    background: #d2d5d9;
+}
+
+.feedback-list-item .feedback-indicator {
+    background: white;
+}
+
+.issues-container {
+    max-width: 800px;
```

Comment:
A suggestion here is a `width: -14px;` should be good enough to close the gap at the bottom of the container to get it closer to the design. That's the quickest fix I'm seeing.

Reply:
A bit confused here. What does a negative width value mean?

Reply:
Apologies there! It should have been `margin-bottom: -14px` but please take a look at playing around with that value for the best fit!